kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -4451,6 +4462,21 @@ private void checkForCustomInput() {
         }
     }
 
+    public void updateHandoverStatus(Message message) {
+        if (!isHandoverHappened && !TextUtils.isEmpty(message.getMetadata().get("LOCALIZATION_VALUE"))) {
+            isHandoverHappened = true;
+            updateAttachmentOptionsVisibilityAfterHandover();
+        }
+    }
+
+    private void updateAttachmentOptionsVisibilityAfterHandover() {
+        if (isHandoverHappened && alCustomizationSettings.getHideAttachmentOptionsBeforeHandover()) {
+            if (attachmentIconLayout != null) {
+                attachmentIconLayout.setVisibility(VISIBLE);
+            }
+        }
+    }
```

Comment:
Change the naming with bot or bot assignee instead of handover. Did you test the case of reopening the conversation which is already assigned to human (after killing the app)

Reply:
Changed the names accordingly and have test that case.